package.json

```
@@ -13,7 +13,7 @@
     "preview": "vite preview",
     "storybook": "storybook dev -p 6006",
     "build-storybook": "storybook build",
-    "format": "biome format . --write"
+    "format:check": "biome format . --write"
```

@techmannih `:check` scripts shouldn't write, make sure to reference other repos...

```suggestion
    "format": "biome format .",
    "format:check": "biome format . --write"
```